math/big/minus_test.ts

```diff
@@ -0,0 +1,6840 @@
+/*
+ *  big.js v5.2.2
+ *  A small, fast, easy-to-use library for arbitrary-precision decimal arithmetic.
+ *  Copyright (c) 2018 Michael Mclaughlin <M8ch88l@gmail.com>
+ *  https://github.com/MikeMcl/big.js/LICENCE
```

Comment:
This link is 404. Also please state the license "MIT License" in text.

I think this module might be better as an external one (deno.land/x) than in deno_std. The reason is that it's quite large and niche - which is going to slowly down my ability to review it. If you keep the same structure as deno_std, it can potentially be moved here later.

Reply:
In fact, you only need to review the core module. It only has a few hundred lines of code

The test case is basically from [big.js](https://github.com/MikeMcl/big.js)

It contains a large number of test cases to ensure the availability and stability of this package.